src/components/info/location-info.vue

```diff
@@ -0,0 +1,193 @@
+<script setup lang="ts">
+import { computed, ref, watch } from 'vue'
+import { useTranslation } from 'i18next-vue'
+import { storeToRefs } from 'pinia'
+import { useInfoStore } from '@/stores/info.store'
+import { Coordinate } from 'ol/coordinate'
+import useMap from '@/composables/map/map.composable'
+import {
+  getQRUrl,
+  getElevation,
+  getNearestAddress,
+  queryInfo,
+} from '@/services/info/location-info'
+
+import StreetView from '@/components/info/street-view.vue'
+
+const { t } = useTranslation()
+const { locationInfo } = storeToRefs(useInfoStore())
+
+const map = useMap().getOlMap()
+
+const shortUrl = ref()
+const qrUrl = ref()
+const elevation = ref()
+const address = ref()
+const clickCoordinateLuref = ref()
+const formatted_coordinates = ref({})
```

Comment:
We do use camel case in our code since the beginning of the project.

Reply:
yep,
do you think it's possible to enforce camelCase in the linter if it is a project standard?

Reply:
yes, `"camelcase": "on"` (I guess, did not test)

https://eslint.org/docs/latest/rules/camelcase#rule-details